--- shapely\geos.py.orig	Fri Dec 12 16:51:59 2014
+++ shapely\geos.py	Fri Dec 12 16:55:37 2014
@@ -84,7 +84,7 @@
         original_path = os.environ['PATH']
         os.environ['PATH'] = "%s;%s;%s" % \
             (egg_dlls, wininst_dlls, original_path)
-        _lgeos = CDLL("geos.dll")
+        _lgeos = CDLL(os.path.join(sys.prefix, "Library", "bin", "geos_c.dll"))
     except (ImportError, WindowsError, OSError):
         raise
 
