diff -Naur src/wafo/MSO.py src/wafo/MSO.py
--- src/wafo/MSO.py	2015-09-08 12:52:55.890913083 -0300
+++ src/wafo/MSO.py	2015-09-08 12:54:54.160918973 -0300
@@ -1,4 +1,4 @@
-# -*- coding: mbcs -*-
+# -*- coding: utf-8 -*-
 # Created by makepy.py version 0.5.00
 # By python version 2.6.2 (r262:71605, Apr 14 2009, 22:40:02) [MSC v.1500 32 bit (Intel)]
 # From type library 'MSO.DLL'
diff -Naur src/wafo/MSPPT.py src/wafo/MSPPT.py
--- src/wafo/MSPPT.py	2015-09-08 12:52:55.890913083 -0300
+++ src/wafo/MSPPT.py	2015-09-08 12:54:46.946918614 -0300
@@ -1,4 +1,4 @@
-# -*- coding: mbcs -*-
+# -*- coding: utf-8 -*-
 # Created by makepy.py version 0.5.00
 # By python version 2.6.2 (r262:71605, Apr 14 2009, 22:40:02) [MSC v.1500 32 bit (Intel)]
 # From type library 'msppt.olb'
