--- include\spatialindex\tools\Tools.h.orig	2016-04-09 14:41:05.660674700 +1000
+++ include\spatialindex\tools\Tools.h	2016-04-09 15:28:33.439971600 +1000
@@ -28,7 +28,7 @@
 #pragma once
 
 
-#if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && !defined __GNUC__
+#if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && (defined _MSC_VER) && (_MSC_VER < 1900) && !defined __GNUC__
   typedef __int8 int8_t;
   typedef __int16 int16_t;
   typedef __int32 int32_t;
@@ -46,7 +46,7 @@
 #endif
 
 #if (defined _WIN32 || defined _WIN64 || defined WIN32 || defined WIN64) && !defined __GNUC__
-  #ifdef SPATIALINDEX_CREATE_DLL
+  #ifdef SIDX_DLL_EXPORT
     #define SIDX_DLL __declspec(dllexport)
   #else
     #define SIDX_DLL __declspec(dllimport)
