diff -Naur httpretty/__init__.py httpretty/__init__.py
--- httpretty/__init__.py	2015-05-18 15:10:10.000000000 -0300
+++ httpretty/__init__.py	2015-09-16 14:17:44.251587140 -0300
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
diff -Naur README.rst README.rst
--- README.rst	2015-05-11 14:39:49.000000000 -0300
+++ README.rst	2015-09-16 14:19:08.298591326 -0300
@@ -511,7 +511,7 @@
 ::
 
     <HTTPretty - HTTP client mock for Python>
-    Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+    Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 
     Permission is hereby granted, free of charge, to any person
     obtaining a copy of this software and associated documentation
