--- shapely/libgeos.py	2015-08-23 11:23:35.000000000 -0300
+++ shapely/libgeos.py	2015-08-29 18:11:53.458018649 -0300
@@ -183,9 +183,10 @@
                 os.path.join(os.path.dirname(__file__), "DLLs"))
             wininst_dlls = os.path.abspath(os.__file__ + "../../../DLLs")
             original_path = os.environ['PATH']
-            os.environ['PATH'] = "%s;%s;%s" % \
-                (egg_dlls, wininst_dlls, original_path)
-            lgeos = CDLL("geos.dll")
+            os.environ['PATH'] = "%s;%s;%s;%s" % \
+                    (egg_dlls, wininst_dlls, original_path,
+                     os.path.join(sys.prefix, "Library", "bin"))
+            lgeos = CDLL("geos_c.dll")
         except (ImportError, WindowsError, OSError):
             raise
 
