--- shapely/libgeos.py	2015-08-23 11:23:35.000000000 -0300
+++ shapely/libgeos.py	2015-08-29 18:15:16.753028774 -0300
@@ -152,25 +152,7 @@
     free.restype = None
 
 elif sys.platform == 'darwin':
-    if not lgeos:
-        # First test to see if this is a delocated wheel with a GEOS dylib.
-        geos_whl_dylib = os.path.abspath(os.path.join(
-            os.path.dirname(__file__), '.dylibs/libgeos_c.1.dylib'))
-        if os.path.exists(geos_whl_dylib):
-            lgeos = CDLL(geos_whl_dylib)
-        else:
-            if hasattr(sys, 'frozen'):
-                # .app file from py2app
-                alt_paths = [os.path.join(os.environ['RESOURCEPATH'],
-                             '..', 'Frameworks', 'libgeos_c.dylib')]
-            else:
-                alt_paths = [
-                    # The Framework build from Kyng Chaos
-                    "/Library/Frameworks/GEOS.framework/Versions/Current/GEOS",
-                    # macports
-                    '/opt/local/lib/libgeos_c.dylib',
-                ]
-            lgeos = load_dll('geos_c', fallbacks=alt_paths)
+    lgeos = CDLL(os.path.join(sys.prefix, 'lib', 'libgeos_c.dylib'))
 
     free = load_dll('c', fallbacks=['/usr/lib/libc.dylib']).free
     free.argtypes = [c_void_p]
